src/services/createIsExternal.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { expect, test } from 'vitest'
+import { createIsExternal } from '@/services/createIsExternal'
+
+test('given undefined host, returns false', () => {
```

Comment:
This test seems backwards to me. If host is undefined. Any url with a host (anything non relative) should be considered external right?

Reply:
I guess my thinking is if you don't supply the context we need to determine if it's external, we always default to the more likely case. Which IMO is that it's not external. Being external also has deeper implications on route changes so I figured it was better to let the router try to handle it like any other route.

Reply:
synced up on this, agreed to change to defaulting to external because presence of error is more helpful than going to not found